.github/workflows/cicd-pipeline.yml

```
@@ -0,0 +1,279 @@
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+        cache: 'npm'
+        
+    - name: Setup PowerShell
+      uses: microsoft/setup-msbuild@v1.3
+      
+    - name: Check environment
+      id: check-env
+      shell: pwsh
+      run: |
```

actionlint (1.7.10)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)